src/index.html

```diff
@@ -11,7 +11,161 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <p>
+          <label for="surname">Surname</label>
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+            autocomplete="off"
+          >
+        </p>
+        <p>
+          <label for="name">Name</label>
+          <input
+            type="text"
+            id="name"
+            name="name"
+            minlength="1"
+            maxlength="35"
+            autocomplete="off"
+          >
+        </p>
+        <p>
+          <label for="howOld">How old are you?</label>
```

Comment:
лучше выносить текст на отдельную строку
```suggestion
          <label for="howOld">
            How old are you?
          </label>
```